docker/local/polygon-edge.sh

```diff
@@ -2,13 +2,6 @@
 
 set -e
 
-# Check if jq is installed. If not exit and inform user.
-if ! command -v jq >/dev/null 2>&1; then
-  echo "The jq utility is not installed or is not in the PATH. Please install it and run the script again."
-  exit 1
-fi
-
-
```

Comment:
Why have you removed this check? Does it not make sense when run in a Docker environment?

Reply:
It does not make sense because this script runs inside docker container that already has JQ according to the docker file.